adwords_downloader/config.py

```diff
@@ -43,11 +43,14 @@ def redownload_window() -> str:
     """The number of days for which the performance data will be redownloaded"""
     return '30'
 
+def output_file_version() -> str:
+    """A suffix that is added to output files, denoting a version of the data format"""
+    return 'v3'
```

Comment:
Shouldn't this default to 'v1'? At least on the package level. If a user requires to have custom version he should specify that in his project.

Reply:
Well, we discussed it for a while: `v3` is best because we have a lot `v2`s elsewhere. And versioning will be a mix of package and project level: So if you change something in the downloader at project level, you'll add a `.1` (it will become `v3.1`). If in the meantime the package format changes, that will become `4.1`, if the project part changes, it will be `3.2`.

Reply:
IMO v3 only makes sense if this the the third version of the *default* query. If so, at least I would love to see a comment here (because the first two versions are not in this git repo): "v1 and v2 were used be the internal predecessor package".

If it is still the first version of the default query, then I do not understand v3: if a venture has a v2 then this means they have an other-than-default query and need to adjust the query and version here as well. Using v3 as a default here will just mean they are lucky they do not need to adjust the version in this cases but will need when they do the next upgrade. But then they have a custom query but not an overwritten version and I can see that this will result in someone trying to change only the query because there is no custom version in the config. IMO it's better to clearly say that "if you adjust the query, you also need to configure a different version".

Re vx.x is there really a case where you would mix package level stuff with project level one? As far as I understand you would overwrite the complete Query and then you are completely on your own versioning schema. So it's more a `k24v3` and not `v<whatever_package_level>.3` because the package level does not matter when you have your own query.